generators/dart/text.js

```diff
@@ -284,7 +290,7 @@ Blockly.Dart['text_prompt_ext'] = function(block) {
     var msg = Blockly.Dart.valueToCode(block, 'TEXT',
         Blockly.Dart.ORDER_NONE) || '\'\'';
   }
-  var code = 'Html.window.prompt(' + msg + ', \'\')';
+  var code = functionName + '(' + msg + ', \'\')';
```

Comment:
The second argument is unused and creates an error.

Reply:
Fixed!